The-SMIRNOFF-force-field-format.md

```diff
@@ -218,6 +216,13 @@ For example, to ensure water molecules are assigned partial charges for [TIP3P](
 </LibraryCharges>
 ```
 
+
+| LibraryCharges section tag version | Tag attributes and default values    | Required parameter attributes   | Optional parameter attributes |
+|------------------------------------|--------------------------------------|---------------------------------|-------------------------------|
+| 0.3                                |                                      | `smirks`, `charges` (indexed)   | `name`, `id`, `parent_id`     |
+
+
+
 ### `<ChargeIncrementModel>`: Small molecule and fragment charges
```

Comment:
Covered in previous comment -- All sections got versions in the SMIRNOFF 0.3 spec. Since they implicitly "had" versions in previous SMIRNOFF specs, those were assumed to be 0.1 and 0.2. By versioning each section independently, sections can undergo rapid development/improvement, while maintaining their own sensible versioning scheme. But while sections progress rapidly, we won't need to update the _top-level_ SMIRNOFF spec version several times a year.